src/exporter.py

```diff
@@ -114,28 +118,43 @@ def track_queue_length(self):
         # we need to cache queue info in exporter in case all workers are offline
         # so that no worker response to exporter will make active_queues return None
         queues = self.app.control.inspect().active_queues() or {}
+        queue_cache = set()
         for info_list in queues.values():
             for queue_info in info_list:
-                self.queue_cache.add(queue_info["name"])
+                queue_cache.add(queue_info["name"])
+
+        # Check celery queues based on worker separator & priority steps
+        separator = '\x06\x16'
+        if 'sep' in self.app.conf["broker_transport_options"]:
+            separator = self.app.conf["broker_transport_options"]['sep']
+        if 'priority_steps' in self.app.conf["broker_transport_options"]:
+            queue_cache_copy = queue_cache.copy()
+            for queue in queue_cache_copy:
+                for step in self.app.conf['broker_transport_options']['priority_steps']:
+                    queue_cache.add(f'{queue}{separator}{str(step)}')
```

Comment:
Where can I find the documentation or source code for these separators?

Reply:
@danihodovic 
https://docs.celeryq.dev/en/stable/userguide/routing.html#redis-message-priorities